components/event-sources/test/fixtures/ksvc.json

```diff
@@ -30,6 +30,9 @@
     "spec": {
         "template": {
             "metadata": {
+                "labels": {
```

Comment:
(nit) No big deal but if you `kubectl get` that object again, it will move `labels` _after_ `annotations` due to the alphabetical order. We should respect that.